src/services/logrocket.ts

```diff
@@ -0,0 +1,41 @@
+import { User } from '@supabase/supabase-js';
+import LogRocket from 'logrocket';
+import setupLogRocketReact from 'logrocket-react';
+import { getUserDetails } from 'services/supabase';
+import { getLogRocketSettings } from 'utils/env-utils';
+
+// Based on node_modules/logrocket/dist/types.d.ts
+interface IUserTraits {
+    [propName: string]: string | number | boolean;
+}
+
+const logRocketSettings = getLogRocketSettings();
+
+export const initLogRocket = () => {
+    if (logRocketSettings.appID) {
+        LogRocket.init(logRocketSettings.appID, {
+            dom: {
+                inputSanitizer: logRocketSettings.sanitize.inputs,
+                textSanitizer: logRocketSettings.sanitize.text,
+            },
+        });
+        setupLogRocketReact(LogRocket);
+    }
+};
```

Comment:
Startup LogRocket. If there is no appID then we assume the functionality is disabled. The `dom` setting is where you can control what is shown or hidden.